cmd/kthena-controller-manager/main.go

```diff
@@ -67,7 +69,12 @@ func main() {
 	pflag.BoolVar(&cc.EnableLeaderElection, "leader-elect", false, "Enable leader election for controller. "+
 		"Enabling this will ensure there is only one active controller. Default is false.")
 	pflag.IntVar(&cc.Workers, "workers", 5, "number of workers to run. Default is 5")
+	pflag.StringVar(&controllers, "controllers", "*", "Comma-separated list of controllers to enable. Available controllers: `modelserving`, `modelbooster`, `autoscaler`. "+
```

Comment:
This description does not reflect the implement well.
 

Without mentioning *, +, -